### PR TITLE
add user sidebar toggle functionaly

### LINKED ISSUE #6: Return to home page button
As a user, i should be able to return to home page from single posting detail page

### LINKED ISSUE #24: User sidebar toggle button
Create a user sidebar toggle button that allows us to view and hide the user sidebar

### PR DESCRIPTION
**Pull request by coopterrones**
#### What's this PR do?
-adds sidebar toggle functionality to user component 
-adds back button to postings page 

#### Where should the reviewer start?
-`PostingView.js` line 76
-`User.js` line 8

#### How should this be manually tested?
-test that the functionality of back button and toggle of sidebar is working without problems

### CODE REVIEW

File: src/Components/User/index.js [hunk 1]
```diff
@@ -1,58 +1,72 @@
...
-    </section>
+    <div className="components-wrapper">
+      <section className={userSidebarClass} id={id}>
+        <div
+          className="profile-picture"
+          style={{
+            backgroundImage: `url(${profilePicture})`,
+            backgroundPosition: "center",
+            backgroundSize: "cover",
+            backgroundRepeat: "no-reapeat",
```
Comment (yiranli624) [inline src/Components/User/index.js:28]: Great work on the side bar and toggle affect!

File: src/Components/User/index.js [hunk 2]
```diff
@@ -1,58 +1,72 @@
...
-  const {
-    id,
-    name,
-    profilePicture,
-    volunteeredHours,
-    upcomingJobs,
-  } = info;
+  const { id, name, profilePicture, volunteeredHours, upcomingJobs } = info;
+
+  const [userOpen, setUserOpen] = useState(true);
```
Comment (okalimbet) [inline src/Components/User/index.js:10]: This is an awesome way to use toggle for the side bar, nice job!

**Review by yiranli624 — APPROVED** [review]
Big improvement on UI! Good job!

**Review by okalimbet — APPROVED** [review]
We need to adjust postings window when sidebar is hidden

**Pull request merged**